### ISSUE TITLE
How about using the Mayaflash Dolphinbar?

### ISSUE THREAD

**Issue by EpicLPer** [initialstate/smart-scale#7]
It always automatically connects when the board is turned on and just costs 20 bucks :)

**Comment by willj2003** [comment]
That's still just a Bluetooth receiver so it would have similar issues with paring

**Comment by EpicLPer** [comment]
No, it's specifically made to emulate a real Wii. When you turn the WiiMote or Board on you don't have to sync it anymore, it'll act like a real Wii at that point and you still have all the sensors, buttons etc. available
